test/mock_batch_test.dart

```diff
@@ -0,0 +1,57 @@
+import 'dart:math';
+import 'package:cloud_firestore_mocks/cloud_firestore_mocks.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() async {
+  final usersPath = "users";
```

Comment:
Please replace all double quotes by single quotes in this file to follow the Analyzer guidelines:

```
   info • Only use double quotes for strings containing single quotes • lib/src/mock_write_batch.dart:39:23 • prefer_single_quotes
   info • Only use double quotes for strings containing single quotes • test/mock_batch_test.dart:6:21 • prefer_single_quotes
   info • Only use double quotes for strings containing single quotes • test/mock_batch_test.dart:10:7 • prefer_single_quotes
   info • Only use double quotes for strings containing single quotes • test/mock_batch_test.dart:11:7 • prefer_single_quotes
   info • Only use double quotes for strings containing single quotes • test/mock_batch_test.dart:12:7 • prefer_single_quotes
   info • Only use double quotes for strings containing single quotes • test/mock_batch_test.dart:25:9 • prefer_single_quotes
   info • Only use double quotes for strings containing single quotes • test/mock_batch_test.dart:26:10 • prefer_single_quotes
   info • Only use double quotes for strings containing single quotes • test/mock_batch_test.dart:37:60 • prefer_single_quotes
   info • Only use double quotes for strings containing single quotes • test/mock_batch_test.dart:46:10 • prefer_single_quotes
   info • Only use double quotes for strings containing single quotes • test/mock_batch_test.dart:50:74 • prefer_single_quotes
```